Makefile

```diff
@@ -203,7 +203,13 @@ GOOS= GOARCH= GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
 }
 endef
 
-deploy-kind: copy-kind-patches docker-build manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+deploy-managed: docker-xplatform-build ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
On second thought, we should probably add `manifests` and `kustomize` as build dependencies.

Reply:
Thank you for the catch!!